samples/spring-security-xsuaa-usage/src/main/java/sample/spring/xsuaa/SecurityConfiguration.java

```diff
@@ -53,6 +55,7 @@ protected void configure(HttpSecurity http) throws Exception {
 				.anyRequest().denyAll()
 			.and()
 				.oauth2ResourceServer()
+				.bearerTokenResolver(new IasXsuaaExchangeBroker(xsuaaServiceConfiguration, new XsuaaOAuth2TokenService()))
```

Comment:
I would keep the sample with the xsuaaServiceConfiguration parameter (most simple use case)

Reply:
with the DefaultOAuth2TokenService it fails to load application context 😕

Reply:
ahhh... okay because of missing Apache Rest Client... BUT then, we have to adapt our default as well, and then again its not needed...

Reply:
Sorry, already merged your PR... could you do that in another PR?